src/main/java/io/openliberty/tools/common/plugins/config/ServerConfigDocument.java

```diff
@@ -529,6 +528,20 @@ private void parseApplication(Document doc, XPathExpression expression) throws X
         }
     }
 
+    private void checkForSpringBootApplicationNode(Document doc, XPathExpression expression, String nodeValue) throws PluginExecutionException {
+        if(expression.equals(XPATH_SERVER_SPRINGBOOT_APPLICATION)){
+            // checking whether any springBootAppNodeLocation already configured from other server configuration files
+            if(springBootAppNodeLocation.isPresent() && springBootAppNodeDocumentURI.isPresent()){
+                throw new PluginExecutionException(String.format("Found multiple springBootApplication elements are specified in multiple server configuration files [%s, %s]. Only one springBootApplication can be configured per Liberty server.", springBootAppNodeDocumentURI.get(), doc.getDocumentURI()));
```

Comment:
"Found multiple springBootApplication elements specified in the server configuration in files [%s, %s]. Only one springBootApplication can be configured per Liberty server."